src/runtimes/node/in_source_config/index.ts

```diff
@@ -93,6 +94,10 @@ export const findISCDeclarations = (
       config.schedule = configExport.schedule
     }
 
+    if (configExport.method !== undefined) {
+      config.methods = Array.isArray(configExport.method) ? configExport.method : [configExport.method]
```

Comment:
We're also doing the array normalisation in Edge Bundler. Do we have to do it here? And if so, why not the capitalisation part as well?

Reply:
Good catch, yes we have to do that here as well!